edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Docker/RuntimeInfoProvider.cs

```diff
@@ -60,10 +61,22 @@ public async Task<IEnumerable<ModuleRuntimeInfo>> GetModules(CancellationToken c
             Option<ContainerInspectResponse> edgeAgentReponse = await this.GetEdgeAgentContainerAsync();
             edgeAgentReponse.ForEach(e => containerInspectResponses.Add(e));
 
-            List<ModuleRuntimeInfo> modules = containerInspectResponses.Select(c => InspectResponseToModule(c)).ToList();
+            List<ModuleRuntimeInfo> modules = containerInspectResponses.Select(InspectResponseToModule).ToList();
             return modules;
         }
 
+        public Task<Stream> GetModuleLogs(string module, bool follow, Option<int> tail, CancellationToken cancellationToken)
```

Comment:
We may want to add a `LoggingOptions` struct for the `follow` and `tail` fields. I don't think it's necessarily a problem now, but this type of thing does have a tendency to introduce parameter bloat.

Reply:
Yes, I agree. I am adding more such types in the actual logs processing. Will use it here if needed as well.